spam/03_spam_data_slicing_tutorial.py

```diff
@@ -330,15 +214,18 @@ def textblob_polarity(x):
 
 
 # %% [markdown]
-# Like we saw above, we'd like to visualize examples in the slice.
-# In this case, most examples with high-polarity sentiments are strong opinions about the video — hence, they are usually relevant to the video, and the corresponding labels are $0$.
+# Again, we'd like to visualize examples in a particular slice. This time, we'll inspect the `textblob_polarity` slice.
+#
+# Most examples with high-polarity sentiments are strong opinions about the video — hence, they are usually relevant to the video, and the corresponding labels are $0$.
+# We define a slice here for *product and marketing reasons*, it's important to make sure that we don't misclassify very positive comments from good users.
```

Comment:
Add "we might" (don't want to imply this is a super real tutorial / we know what we're talking about"